website/oldsite/home.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Get rid of all the oldsite stuff. Seems like you guys have it all copied over into the new one anyways.